website/blog/2024-03-18-deploying-offline-with-docker/index.md

```diff
@@ -14,39 +14,54 @@ tags: [deployment offline]
 Are you facing a situation where your work environment lacks internet access, and you're wondering if you can still deploy Tabby? Fear not, because the answer is YES! 🐱📣
 
 ## Prerequisite📋
+
 * Docker installed on both the internet-connected computer and the offline computer.
 
 ## The Game Plan🚀
+
 * Create a Docker image on a computer with internet access. 
 * Transfer the image to your offline computer.
 * Run the Docker image and let Tabby work its magic! ✨
 
 ## Step-by-Step Guide🐾
+
 1. Create a new **Dockerfile** on a computer with internet access.
+
 ```docker
 FROM tabbyml/tabby
 
-RUN /opt/tabby/bin/tabby-cpu download --model TabbyML/StarCoder-1B
+ENV TABBY_MODEL_CACHE_ROOT=/models
+
+RUN /opt/tabby/bin/tabby-cpu download --model StarCoder-1B
 ```
+
```

Comment:
add some explaination here on what does TABBY_MODEL_CACHE_ROOT do and what does this command do

Reply:
Added the following:

The **TABBY_MODEL_CACHE_ROOT** env var sets the directory for saving downloaded models. Setting it to `/models` will store the model in the `/models` directory of the Docker container.

Reply:
could you also explain the command? it downloads the model to the dir.

Reply:
Changed to:

The **TABBY_MODEL_CACHE_ROOT** env var sets the directory for saving downloaded models. By setting `ENV TABBY_MODEL_CACHE_ROOT=/models`, we instruct Tabby to save the downloaded model files in the `/models` directory within the Docker container during the build process.